package.json

```diff
@@ -78,7 +81,7 @@
     "power-assert": "^1.6.0",
     "protobufjs": "^6.8.6",
     "proxyquire": "^2.0.1",
-    "typescript": "^2.9.2",
-    "gts": "^0.7.1"
+    "ts-node": "^7.0.0",
```

Comment:
Where/how are you using tsnode?

Reply:
This allows me to run the TypeScript code in-place from within IntelliJ. 
It's also possible to just install this globally and not add `ts-node` to the dev-dependencies. 
I was debating committing the IntelliJ test runners as well, though, similar to what we have done in the main Firestore repo (https://github.com/firebase/firebase-js-sdk/tree/master/packages/firestore/.idea/runConfigurations). These test runner rely on `ts-node` as an initialization step.